tests/terraform/checks/resource/aws/test_TransferServerIsPublic.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from checkov.common.models.enums import CheckResult
+from checkov.terraform.checks.resource.aws.TransferServerIsPublic import check
+import hcl2
+
+
+class TestTransferServerIsPublic(unittest.TestCase):
```

Comment:
Please change the tests to use the newer set up like here #1182

And also please add a test without specifying the `endpoint_type` should result in a failure.